sk-learn-analysis/myst.yml

```
@@ -6,7 +6,7 @@ project:
   # description:
   # keywords: []
   # authors: []
-  github: https://github.com/jupyter-book/mystmd-quickstart
+  github: https://github.com/Auslum/sklearn-survey-report-2024
```

```suggestion
  github: https://github.com/sklearn-survey-report-2024/sklearn-survey-report-2024
```
Let's link to the main repo.